OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -527,7 +527,7 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 					if (depth == 2 && parentKey.StartsWith("DeathSounds") && node.Key == "InfDeaths")
 						node.Key = "DeathTypes";
 
-					if (depth == 2 && parentKey == "SpawnsViceroid" && node.Key == "InfDeath")
+					if (depth == 2 && parentKey == "SpawnViceroid" && node.Key == "InfDeath")
```

Comment:
Commit that introduced this: https://github.com/OpenRA/OpenRA/commit/c17b284051aa1168eac4a7685f53bad1b434792b
[SpawnViceroid.cs on this commit.](https://github.com/OpenRA/OpenRA/blob/c17b284051aa1168eac4a7685f53bad1b434792b/OpenRA.Mods.Cnc/SpawnViceroid.cs)